lib/fluent/plugin/storage_local.rb

```diff
@@ -29,10 +30,10 @@ class LocalStorage < Storage
       DEFAULT_FILE_MODE = 0644
```

Comment:
`DEFAULT_FILE_MODE` and `DEFAULT_DIR_MODE` aren't used anymore, right?

Reply:
Yes, I've removed it.